verification/tests/server.rs

```diff
@@ -0,0 +1,43 @@
+use std::num::NonZeroUsize;
+
+use pretty_assertions::assert_eq;
+use verification::{make_retrying_request, run_http_server, Config};
+
+#[actix_rt::test]
+async fn server_start() {
+    let mut config = Config::default();
+    config.solidity.enabled = false;
+    config.sourcify.enabled = false;
+    config.metrics.enabled = true;
+    let base = format!("http://{}", config.server.addr);
+    let metrics_base = format!("http://{}", config.metrics.addr);
+    let _server_handle = {
+        let config = config.clone();
+        tokio::spawn(async move { run_http_server(config).await })
+    };
+
+    let sleep_between = Some(tokio::time::Duration::from_millis(100));
+    let attempts = NonZeroUsize::new(20).unwrap();
```

Comment:
let's make this high (`100` == 10 secs), so it'll be like a timeout, and not some arbitrary "well this probably takes something around 2 secs on my machine may be" :)